<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerformanceSetup</class>
 <widget class="QWidget" name="PerformanceSetup">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>642</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设置</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>201</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>警戒值设置</string>
   </property>
   <widget class="QLineEdit" name="memLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cpuLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="batteryLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>内存(M)：</string>
    </property>
   </widget>
   <widget class="QLabel" name="cpuTempLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU(%)：</string>
    </property>
   </widget>
   <widget class="QLabel" name="wifiLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>电池温度(℃):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="batteryLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>120</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="wifiLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="mobileLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="batteryLabel_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wifi流量(M):</string>
    </property>
   </widget>
   <widget class="QLabel" name="cpuTempLabel_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Mobile流量(M):</string>
    </property>
   </widget>
   <widget class="QLabel" name="mobileLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU温度(℃):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cpuTempLineEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>显示间隔(s)</string>
   </property>
   <widget class="QLineEdit" name="timeLineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>361</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>统计点</string>
   </property>
   <widget class="QRadioButton" name="debugRBtn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>debug</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="releaseRBtn">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>release</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>54</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>APP版本：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IPLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="portLineEdit">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeWidget" name="xmlTreeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>341</width>
      <height>191</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="selectXmlBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>341</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>点击选择XML文件</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="statCBox">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>310</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>统计点测试</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
